src/rez/cli/rm.py

```diff
@@ -15,6 +15,11 @@ def setup_parser(parser, completions=False):
         "-p", "--package",
         help="remove the specified package (eg 'foo-1.2.3'). This will work "
         "even if the package is currently ignored.")
+    group.add_argument(
+        "-f", "--family",
+        help="remove the entire specified package family (eg 'python'). This "
```

Comment:
the word "entire" is a bit confusing here. "remove the entire", but, "only if its empty"

Reply:
agreed